code/__HELPERS/type2type.dm

```diff
@@ -396,7 +396,22 @@ for(var/t in test_times)
 	return
 
 
-
+//for each body zone, gives the corresponding body_parts_covered bitfield
+/proc/body_zone2body_parts_covered(b_zone)
+	b_zone = check_zone(b_zone)
+	switch(b_zone)
+		if("chest")
+			return CHEST
+		if("head")
+			return HEAD
+		if("r_arm")
+			return ARM_RIGHT
+		if("l_arm")
+			return ARM_LEFT
+		if("r_leg")
+			return LEG_RIGHT
+		if("l_leg")
+			return LEG_LEFT

```

Comment:
This is used by damage_clothes() to get which clothes are on the area we're hitting.